examples/04_glm_first_level/plot_two_runs_model.py

```diff
@@ -56,16 +56,40 @@
 
 design_matrices = [data["design_matrix1"], data["design_matrix2"]]
 
-
 # %%
-# Initialize and run the GLM
-# --------------------------
+# Specify the model and inspect it
+# --------------------------------
 # First, we need to specify the model
 # before fitting it to the data.
+#
+# We can specify the model with fitting it
+# by using the ``design_only=True`` parameter.
+#
 # Note that a brain mask was provided in the dataset,
 # so that is what we will use.
+#
+# We can inspect the model by generating a report.
+#
 from nilearn.glm.first_level import FirstLevelModel
 
+fmri_glm = FirstLevelModel(
+    mask_img=data["mask"],
+    smoothing_fwhm=5,
+    minimize_memory=False,
+    design_only=True,
+)
+report = fmri_glm.generate_report()
+# report.open_in_browser()
+
+# This report can be viewed in a notebook.
+report
```

Comment:
the report does not look as expected: no design matrix.
also if a mask_img or some input data was passed, it should at least show the mask that will be used for the GLM